source/clear-linux/tutorials/kubernetes/kubernetes-bp.rst

```diff
@@ -0,0 +1,104 @@
+.. _kubernetes-bp:
+
+Kubernetes Best Practices on |CL|
+#################################
+
+This tutorial shows you how to manage your Kubernetes cluster while using
+:command:`swupd` to update |CL-ATTR|.
+
+In our tutorial :ref:`kubernetes`, we explain how to set up a Kubernetes
+cluster on |CL| using `kubeadm`. `Kubeadm documentation`_ often builds on the
+assumption that the distribution uses a traditional package manager (e.g.,
+RPM/DEB). In contrast, |CL| manages packages in bundles. This document
+describes best practices to manage cluster upgrades with kubeadm on a |CL|-based cluster.
+
```

Comment:
In general, I think we should add a short example, with a master and worker nodes, to show how to use swupd udpate with stdout.